pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/integrationLayer/ContentList.kt

```diff
@@ -54,10 +63,11 @@ sealed interface ContentList {
             return "$RootRoute/latestMediaByShow/$urn"
         }
 
-        companion object {
-            const val route = "$RootRoute/latestMediaByShow/{showUrn}"
+        companion object : ContentListFactory<LatestMediaForShow> {
+            override val route = "$RootRoute/latestMediaByShow/{showUrn}"
+            override val trackerTitle = "Latest media for show"
```

Comment:
For the `trackerTitle` values, we sometimes observe uppercase starting letter, and sometimes not. Is this intentional?